apps/user-service/src/schemas/user.schema.ts

```diff
@@ -0,0 +1,26 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import { v4 as uuidv4 } from 'uuid';
+@Schema({ timestamps: true })
```

Comment:
new line?

Reply:
Yes, timestamp's used to generate createdAt/updatedAt values

Reply:
```suggestion
import { v4 as uuidv4 } from 'uuid';
@Schema({ timestamps: true })
```

why here not new line?